samples/bot-release-management/csharp/README.md

```diff
@@ -42,7 +42,7 @@ This is a sample application which demonstrates how to create a service hooks on
 1) Run ngrok - point to port 3978
 
 ```bash
-# ngrok http -host-header=rewrite 3978
+# ngrok http 3978 --host-header="localhost:3978"
 ```
 2) Once started you should see ngrok URL like `https://xxxxx-xxx-e125.ngrok.io`. Copy it, this is your baseUrl that will used as endpoint for Azure bot and webhook.
```

Comment:
https://xxxxx-xxx-e125.ngrok**.io**
This format of URL should also be changed at all the places and in all the changes files in this PR like below:
**https://xxx-xxx-xxx.ngrok-free.app**

Reply:
Please check manifest file as well

Reply:
Hello @Wajeed-msft - Could you please confirm whether we need to do above changes as well?